modules/microshift-custom-ca-proc.adoc

```diff
@@ -6,14 +6,18 @@
 [id="microshift-custom-cas-configuring_{context}"]
 = Configuring custom certificate authorities
 
-To configure externally generated certificates and domain names using custom certificate authorities (CAs), add them to the {microshift-short} `/etc/microshift/config.yaml` configuration file. You must also configure the host operating system trust root.
+To configure externally generated certificates and domain names by using custom certificate authorities (CAs), add them to the {microshift-short} `/etc/microshift/config.yaml` configuration file. You must also configure the host operating system trust root.
+//Q: is this procedure still valid, or should we use
```

Comment:
Yes its still valid, its concerns the API server and the linked procedure are for the ingress-controller , those are two different configuration.